doc/manuals/admin/crate.md

```diff
@@ -0,0 +1,9 @@
+# Crate
+
+[**Crate**](https://crate.io) is Quantumleap's default backend where NGSI data will be persisted. Until Quantumleap implements its querying API, you can interact directly with Crate to query all the data QL has saved from the received notifications.
```

Comment:
[**Crate**](https://crate.io) is QuantumLeap's default backend where NGSI data will be persisted. Until QuantumLeap implements its querying API, you can interact directly with Crate to query all the data QuantumLeap has stored from the received notifications.